test/serializer/additional-functions/Example1.js

```diff
@@ -0,0 +1,47 @@
+// additional functions
```

Comment:
Just to confirm: this PR did not add any visible changes, right? So these two testcases are not related to this PR -- they should pass even without this PR?

Reply:
No visible changes, I probably could've put the tests in a different PR.